coding-standard/action.yml

```diff
@@ -0,0 +1,120 @@
+name: "Coding Standard"
+author: "Graycore"
+description: "A Github Action that runs the Magento Coding Standard."
+
+inputs:
+  php_version:
+    required: true
+    default: "8.1"
+    description: "PHP version used to do the coding standard check."
+
+  composer_version:
+    required: true
+    default: "2"
+    description: "The version of composer to use."
+
+  path:
+    required: true
+    default: "app/code"
+    description: "The directory (relative to the project root) in which the coding standard will be checked. Used when the event is not a pull request."
+
+  version:
+    required: false
+    default: "31"
+    description: "The version of the coding standard to use. If not provided, will use the latest version."
+
+  severity:
+    required: false
+    default: "5"
+    description: "The minimum severity required to display an error or warning (default: 5)"
+
+  warning_severity:
+    required: false
+    default: "5"
+    description: "The minimum severity required to display a warning"
+
+  error_severity:
+    required: false
+    default: "5"
+    description: "The minimum severity required to display an error"
+
+runs:
+  using: composite
+  steps:
+    - name: Checkout Project
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+        path: standard
+
+    - name: Set PHP Version
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: ${{ inputs.php_version }}
+        tools: composer:v${{ inputs.composer_version }}
+        coverage: none
+
+    - name: Install Coding Standard && Codesniffer baseline
+      shell: bash
+      working-directory: standard
+      run: |
+        git config --global advice.detachedHead false
+        composer require "magento/magento-coding-standard"
```

Comment:
We need to use `inputs.version` here like in https://github.com/graycoreio/github-actions-magento2/blob/81a1eb2273864842286a4427519c9d29146c5ac8/coding-standard/action.yml#L63, right?